### DISCUSSION TITLE
GTerminal - Funny addo

### DISCUSSION THREAD

**Discussion by ELPatitoLoco** [TAServers/issues#34] [Addon Suggestions]
Just a small addition to have something fun to use.
This addon adds a terminal like entity wich allows you to use a ms-dos /windows 95 like os.
[GTerminal Universal](https://steamcommunity.com/sharedfiles/filedetails/?id=3207879425&searchtext=)

**Comment by Derpius** [comment]
This is what starfall is for
I have an extensible terminal library in my public starfalls repo that I've used in other projects to create a lua and brainfuck REPL

**Reply by ELPatitoLoco** [reply]
Ig yeah, but still fun for users that don't use or know sf.

**Reply by Derpius** [reply]
A niche audience of people who don't know how to program but also want a fake shell to play in

Library is here btw: https://github.com/Derpius/public-starfalls/tree/master/console

**Comment by Derpius** [comment]
Created one of the new addon suggestion issues from this: https://github.com/TAServers/issues/issues/36
